src/cycax/cycad/Cuboid.py

```diff
@@ -0,0 +1,43 @@
+from Part import Part
```

Comment:
We might have to be careful here. FreeCAD also has a part e.g. https://wiki.freecad.org/Part_scripting

CycadPart might be safer.